content/blog/2024-06-15-rustnl-2024-unconference.md

```diff
@@ -0,0 +1,203 @@
+
+Some people also wanted Parley to be decoupled from Swash.
+
+People were especially interested in the idea of having access to a common text-editing widget, or at least infrastructure for creating one.
+That widget would handle IME, text selection, accessibility actions, etc, in a way that would feel native to different platforms with their own text-editing quirks (mobile, macOS, Linux with vim mode, etc).
+
+### Winit
+
+By now all of the Rust ecosystem has firmly converged on [`winit`](https://github.com/rust-windowing/winit) as the windowing platform-abstraction solution of choice.
+
+(Well, not all! [One small project](https://github.com/makepad/makepad) with indomitable maintainers still holds out against the invaders.)
+
+The consensus during this discussion seemed to be that Winit was there to stay.
+[TAO](https://github.com/tauri-apps/tao) users present were generally interested in being able to use upstream Winit in the future, if its API adopts or supersedes the improvements made in TAO.
+Most were happy to see Winit becoming more trait-oriented, and to see the `dpi` crate spun off from Winit.
+
+Glazier is still shelved and we're planning to port its features to Winit until it reaches feature parity.
+
+A discussion that came up was the idea of splitting more "vocabulary types" from Winit into standalone crates.
+Vocabulary crates, e.g. crates that only define types and very little code, have a very low cost in memory and build times, while allowing projects to interop with e.g. Winit without actually adding a dependency on Winit.
+
+This was connected with the discussions about text editing: having a vocabulary crate that would list Winit's events and also Parley's cursor movements would allow for a very low-dependency text editing widget.
+
+### Accessibility
+
+*Note: This section is also my personal opinion and doesn't represent the Linebender project either.*
+
+The subject of Accessibility came up informally, but I'll claim it was underexplored.
+
+While the subject was on the agenda, by the time we reached it the Unconference had been running long enough that everyone was tired and ready to leave or get to the catering.
+This is an unfortunate pattern that I think is worth pointing out: if projects don't make a deliberate effort to prioritize discussions of accessibility, the subject will be left out.
+
+Matt Campbell, the maintainer of AccessKit, could not attend the Unconference.
+I'm concerned that, in his absence, other maintainers might have felt a dilution of responsibility: since the accessibility guy wasn't there, nobody felt empowered to bring up accessibility.
+
+(This might sound like it contradicts my section about structurelessness above.
+It doesn't.
+I'm not saying organizers should have pushed harder for accessibility - they did try - I'm saying that we should strive for a culture where framework developers think about accessibility without waiting for someone to tell them to.)
```

Comment:
Definitely agree on this point - I was pleased with how Kevin pushed for Accessibility to be discussed, but am disappointed that it didn't catch.